roles/server/tasks/go-server.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: apt install go-server
   sudo: yes
-  apt: pkg=go-server={{ GOCD_GO_VERSION }} state=present force=yes
+  apt: pkg=go-server{{ '=' + GOCD_GO_VERSION if GOCD_GO_VERSION != 'latest' else ''}} state=present force=yes
```

Comment:
Would you make a similar change for go-server yum installer and the go-agent playbooks?